app-frontend/src/app/components/vectors/vectorImportModal/vectorImportModal.html

```diff
@@ -0,0 +1,145 @@
+<div class="modal-scrollable-body">
+  <div class="modal-header">
+    <button type="button" class="close" aria-label="Close" ng-click="$ctrl.handleClose()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+    <h4 class="modal-title" ng-if="!$ctrl.resolve.project">
+      Upload Shape
+    </h4>
+  </div>
+
+  <!-- Body for SELECT_FILE -->
+  <div class="modal-body" ng-if="$ctrl.currentStepIs('SELECT_FILE')">
+    <div class="content">
+      <h3 class="no-margin">Select a zipped shapefile</h3>
+      <ng-container ng-if="!$ctrl.selectedFile">
+        <p>Select shape files from your computer. {{$ctrl.BUILDCONFIG.APP_NAME}} currently allows for uploading zipped shape files only.</p>
```

Comment:
This text should probably reflect the fact that you can only select a single shapefile. We should also stick to 'shapefile' as a single word, which is the official way they are referred to.

Reply:
Additionally, the text indicates that drag and drop is an option. DnD didn't work for me and I don't see the `ngf-drop-available` property being set.

Reply:
oops